src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿
+@using SFA.DAS.EAS.Portal.Client.Types;
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+@{
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+
+    if (Model.HasSingleClosedVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Closed);
+        actionLinkText = "Manage vacancy";
+        numberOfApplications = vacancy.NumberOfApplications;
+    };
+}
+<h3 class="das-panel__heading">Your vacancy</h3>
+<p>@vacancy.TrainingTitle <span class="das-!-display--block das-text--muted">VAC @vacancy.Reference</span></p>
+<dl class="das-definition-list das-definition-list--inline">
+    <dt class="das-definition-list__title">Closing date</dt>
```

Comment:
From looking at the commit history for this PR, it looks like you've addressed the colon discrepancy between design and acceptance criteria 👍.

Reply:
Checked with Nav, and they should be there. The css class `das-definition-list__title` adds them, so all is good.